code/src/java/pcgen/gui2/PCGenFrame.java

```diff
@@ -939,7 +940,8 @@ boolean showSaveCharacterChooser(CharacterFacade character)
 		File prevFile = file;
 		if (file == null || StringUtils.isEmpty(file.getName()))
 		{
-			file = new File(parentPath, character.getNameRef().get() + Constants.EXTENSION_CHARACTER_FILE);
+			String characterName = FileHelper.sanitizeFilename(character.getNameRef().get());
+			file = new File(parentPath, characterName + Constants.EXTENSION_CHARACTER_FILE);
```

Comment:
Any reason not to do this on the same line?

Reply:
I could do it on the same line. It seemed like the line was getting pretty long, though, and breaking it up looked a bit more clear.